examples/cryptorealtime/src/main/java/source/CryptoMarketTradeUnboundedSource.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2019 Google
+ * <p/>
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Similar comment here as elsewhere about the license header comment format; please update all Java and other files accordingly.

Reply:
all updated https://github.com/GoogleCloudPlatform/professional-services/pull/191/commits/05ab1977716987c886d5bb7bb221ea4c72cdc1ee